xep-0045.xml

```diff
@@ -1385,6 +1385,7 @@
 </presence>
 ]]></example>
       <p>Before attempting to enter the room, a MUC-compliant client SHOULD first discover its reserved room nickname (if any) by following the protocol defined in the <link url='#reservednick'>Discovering Reserved Room Nickname</link> section of this document.</p>
+      <p>When a MUC service receives an &lt;x/&gt; tagged join stanza from an already-joined client (as identified by the client's full JID), the service SHOULD assume that the client lost its synchronization, and therefore it SHOULD send exactly the same stanzas to the client as if it would actually join the MUC. The server MAY also send a presence update to the other participants according to the received join presence.</p>
```

Comment:
I think there is a difference between the two sentences. First, what is the situation:

*the service SHOULD assume that the client lost its synchronization,*

then, how to handle it:

*and therefore it SHOULD send exactly the same stanzas to the client as if it would actually join the MUC*

Maybe it's not a full-fledged rationale, but at least an explanation of why we want that behavior. Anyway, feel free to propose a better wording that's more to the point :)

> What does "actually joined" mean?

The server should send everything to the client which it would send as response if the client had requested a join and hadn't been part of the MUC before. Does that sound better?